src/stores/slices/authSlice.ts

```diff
@@ -3,7 +3,7 @@ import type { PayloadAction } from '@reduxjs/toolkit';
 import { AuthState } from '../../types/auth.types';
 
 const initialState: AuthState = {
-  token: null,
+  token: null || localStorage.getItem('access_token'),
```

Comment:
`localStorage.getItem('access_token')` enough

Reply:
so, remove it please

Reply:
so, remove it please